ci/test/lint-docs.sh

```diff
@@ -15,5 +15,11 @@ set -euo pipefail
 
 git clean -ffdX ci/www/public
 hugo --gc --baseURL https://ci.materialize.com/docs --source doc/user --destination ../../ci/www/public/docs
+# To avoid breaking URLs that already exist, pull the production sitemap
+# and convert it to HTML. htmltest will scan the current prod paths and
+# complain if any have been broken in this build.
+curl https://materialize.com/docs/sitemap.xml -o ci/www/sitemap.xml
```

Comment:
As a rule we try to avoid introducing dependencies on external state in CI (i.e., besides GitHub and DockerHub). Could you change this to do something like:

```
git checkout main
git pull
hugo --gc --source doc/user --destination tmp
sed ... tmp/sitemap.xml >> ci/www/public/index.html
```

Reply:
OK trying this now